cache/fs.go

```diff
@@ -113,6 +115,21 @@ func (fs FSCache) GetArtifact(artifactID string) (types.ArtifactInfo, error) {
 	return info, nil
 }
 
+// DeleteBlob removes blob by Id
+func (fs FSCache) DeleteBlobs(blobIDs []string) error {
+	var errs error
+	fs.db.Update(func(tx *bolt.Tx) error {
+		bucket := tx.Bucket([]byte(blobBucket))
+		for _, blobID := range blobIDs {
+			if err := bucket.Delete([]byte(blobID)); err != nil {
+				errs = multierror.Append(errs, err)
+			}
+		}
+		return nil
+	})
+	return errs
```

Comment:
Also, we should handle an error from `Update`.

```suggestion
	err := fs.db.Update(func(tx *bolt.Tx) error {
		bucket := tx.Bucket([]byte(blobBucket))
		for _, blobID := range blobIDs {
			if err := bucket.Delete([]byte(blobID)); err != nil {
				errs = multierror.Append(errs, err)
			}
		}
		return nil
	})
	if err != nil {
		return xerrors.Errorf("DB update error: %w", err)
	}
	return errs
```

Reply:
done

Reply:
oh @knqyf263 I missed.
is it really possible that `fs.db.Update` will return any error in this case?

Reply:
we accumulate all inner errors into `errs`...

Reply:
Yes, it is possible to return an error. For example, the database is locked and db.Update can't open the database for write operations. The database is broken for some reason. etc...

Reply:
oh, sure. thanks for explanation!